test/test_format_checker.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+CVE-bin-tool Format Checker tests
+"""
+import os
+import re
+from os.path import dirname, join
+from typing import List
+from unittest.mock import call, mock_open, patch
+
+import pytest
+
+import cve_bin_tool.format_checkers as format_checkers
+
+
+class TestFormatCheckers:
+    """Tests the CVE Bin Tool Format Checkers Script"""
+
+    @classmethod
+    def setup_class(cls):
+        cls.mock_path_dir = join(dirname(__file__), "format_checkers")
+        cls.mock_checkers = [
+            "libssh2",
+            "polarssl_fedora",
+            "radare2",
+            "rsyslog",
+            "open_vm_tools",
+            "libssh2",
+            "polarssl_fedora",
+            "busybox",
+            "commons_compress",
+            "bzip2",
+        ]
+        cls.mock_checkers.sort()
+
+    @pytest.mark.asyncio
+    async def test_update_allowed_words(self):
+        """Tests the update_allowed_words function"""
+        file_path = os.path.join(self.mock_path_dir, "allow.txt")
```

Comment:
There are some instances where just `join` is used. Maybe we could stick with either of one?

Reply:
Ok!